pa11y.json

```diff
@@ -0,0 +1,3 @@
+{
+  "hideElements": "iframe"
```

Comment:
When running the server in test/dev environment phoenix adds this iframe to the bottom of the html:
```html
<iframe src="/phoenix/live_reload/frame" style="display: none;"></iframe>
```